lib/controlling/Router.js

```diff
@@ -31,10 +33,79 @@ var Router = Backbone.Router.extend({
         return Promise.reject({
             status: statusCode
         });
+    },
+
+    route: function(route, name, callback) {
```

Comment:
instead of a RouterBrewery and ServerRouter vs Backbone.Router, Brisket.Router implements route method that wires up ServerDispatcher or Backbone.History depending on environment